src/computation/computation.graphql

```diff
@@ -10,4 +10,41 @@ input CourseEntryInput {
 type CourseEntry {
     courseId: String!
     studyProgram: String!
-}
\ No newline at end of file
+}
+
+input CourseRecommendationRequest {
+    variant: String!
+    semesterKey: SemesterKeyInput!
+	selectedCourse: [CourseKeyInput!]! 
```

Comment:
don't mix tabs and spaces
![image](https://user-images.githubusercontent.com/24814968/144999149-dca4a972-9628-4cb4-be0f-891d27b5f9d1.png)
and other various places

Reply:
the tab in github code review is shit